x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/index.tsx

```diff
@@ -22,7 +22,7 @@ import { useHistory } from 'react-router-dom';
 import styled from 'styled-components';
 import { APIReturnType } from '../../../../services/rest/createCallApmApi';
 import { APMError } from '../../../../../typings/es_schemas/ui/apm_error';
-import { IUrlParams } from '../../../../context/UrlParamsContext/types';
+import { IUrlParams } from '../../../../context/url_params_context/types';
```

Comment:
Should we start importing types like this?
`import type { IUrlParams } from '../../../../context/url_params_context/types';`